src/polymer/expression-scanner.ts

```diff
@@ -125,6 +135,67 @@ export class DatabindingExpression {
     return correctSourceRange(
         databindingRelativeSourceRange, this.locationOffset);
   }
+
+  private _extract() {
+    if (this._expressionAst.body.length !== 1) {
+      this.warnings.push(this._validationWarning(
+          'Expected exactly one expression', this._expressionAst));
+      return;
+    }
+    const expressionStatement = this._expressionAst.body[0]!;
+    if (expressionStatement.type !== 'ExpressionStatement') {
+      this.warnings.push(this._validationWarning(
+          'ExpressionStatement expected', expressionStatement));
+      return;
+    }
+    let expression = expressionStatement.expression;
+    if (expression.type === 'UnaryExpression') {
+      if (expression.operator !== '!') {
+        this.warnings.push(this._validationWarning(
+            'Only the not (!) operator is supported.', expression));
+        return;
+      }
+      expression = expression.argument;
+    }
+    this._handleSubExpression(expression);
+  }
+
+  private _handleSubExpression(expression: estree.Node) {
+    if (expression.type === 'Literal') {
+      return;
+    }
+    if (expression.type === 'Identifier') {
+      this.properties.push({
+        name: expression.name,
+        sourceRange: this.sourceRangeForNode(expression)!
+      });
+      return;
+    }
+    if (expression.type === 'MemberExpression') {
+      this._handleSubExpression(expression.object);
+      return;
+    }
+    if (expression.type === 'CallExpression') {
+      this._handleSubExpression(expression.callee);
+      for (const arg of expression.arguments) {
+        this._handleSubExpression(arg);
+      }
+      return;
+    }
+    this.warnings.push(this._validationWarning(
+        `Unexpected node type: ${expression.type
```

Comment:
Changed to a two sentence pattern. Simple first sentence, more technical second sentence.